src/Shared/Program.cs

```diff
@@ -63,7 +67,9 @@ public static void Main(string[] args)
                 }
                 else
                 {
-                    method.Invoke(null, arguments);
+                    var result = method.Invoke(null, arguments);
+
+                    Console.WriteLine($"Result: {result}");
```

Comment:
Do we care about the case where `result` is null? Such as the case where the method returns `void`